### DISCUSSION TITLE
为什么制作一个服务器启动器

### DISCUSSION THREAD

**Discussion by BaiBai-1020** [Meloong-Git/PCL#803] [提问] [answered]
功能有：**下载**各版本**服务端**，添加**插件**、**MOD**、**整合包**；客户端地图与服务端**地图转换**；**修改server.properties**；**修改登入方式**；**启动**服务端；**修改**服务器**图标**（server-icon.png）等

**Comment by WForst-Breeze** [comment, marked as answer]
没看懂您想问什么……我就按照我的理解来了
PCL2 是 MC **启动器**，而非开服工具，您提及的功能远超出了启动器的服务范畴（您这条 Discussion 中提及的内容涉及了太多已经被拒绝的 Issues 了，就不一一列举了
不过 PCL2 可能在未来提供**原版服务端**的**下载服务**，请见 #113
